src/components/PDFView/index.js

```diff
@@ -148,6 +148,7 @@ class PDFView extends Component {
                                 width={pageWidth}
                                 key={`page_${index + 1}`}
                                 pageNumber={index + 1}
+                                loading=""
```

Comment:
Can we add some explanation here or something please? If I saw this here there is nothing that tells me passing the empty string is intentional.

Reply:
It would be nice but no pressure on you @Prince-Mendiratta. We merged it without this suggestion so the comment is for @aimane-chnaif and @danieldoglas. I really think the root cause of the issue in something to do with the PDF library. If we implement a workaround the least we can do is document with a code comment why it exists.

Reply:
Thanks !